components/new-pedal-form/index.js

```diff
@@ -6,35 +6,34 @@ import { v4 as uuidv4 } from "uuid";
 import {
   StyledButtonContainer,
   StyledDimension,
-  StyledDimensionInput,
   StyledDimensionsWrapper,
   StyledFormContainer,
   StyledInput,
   StyledLabel,
   StyledStereoWrapper,
 } from "./new-pedal-form.styled";
-import { StyledDimensions } from "../pedal-details-specs/pedal-details-specs.styled";
 
 export default function NewPedalForm() {
   const [name, setName] = useState("");
   const [manufacturer, setManufacturer] = useState("");
   const [madeIn, setMadeIn] = useState("");
-  const [width, setWidth] = useState(0);
-  const [depth, setDepth] = useState(0);
-  const [height, setHeight] = useState(0);
+  const [width, setWidth] = useState("");
+  const [depth, setDepth] = useState("");
+  const [height, setHeight] = useState("");
   const [stereo, setStereo] = useState(false);
   const [tags, setTags] = useState([]);
-  const [selectedCategory, setSelectedCategory] = useState("");
+  const [selectedCategory, setSelectedCategory] = useState([]);
```

Comment:
for a further idea, (direct editing of just one part of the object by doubleclick in detail-view without having to open the whole form) i thought it would be handy if they have their own state...